invenio_rdm_migrator/load/postgresql.py

```diff
@@ -43,11 +43,19 @@ def as_csv_row(dc):
 class PostgreSQLCopyLoad(Load):
     """PostgreSQL COPY load."""
 
-    def __init__(self, db_uri, table_generators, tmp_dir):
+    def __init__(
+        self, db_uri, table_generators, tmp_dir, existing_data=False, **kwargs
+    ):
         """Constructor."""
         self.db_uri = db_uri
-        self.tmp_dir = Path(tmp_dir) / f"tables-{ts(fmt='%Y-%m-%dT%H%M%S')}"
         self.table_generators = table_generators
+        self.tmp_dir = Path(tmp_dir)
```

Comment:
if loading from existing data, there is no need for `tables-ts` appended since they should be in the root of tmp